template/.github/workflows/build.yml.j2

```diff
@@ -16,6 +16,9 @@ on:
       - '[0-9][0-9].[0-9]+.[0-9]+'
   pull_request:
   merge_group:
+  schedule:
```

Comment:
If possible I'd like to have a comment here for those like me that are crontabwise challenged. I can never remember the syntax.

Can we have a comment saying that this runs every saturday night?

Reply:
Or a link to the crontabguru explanation (or is that too much indirection?)

https://crontab.guru/#15_3_*_*_6

Reply:
Either is fine for me I think